samples/msal-core-samples/VanillaJSTestApp/app/ie11-sample/README.md

```diff
@@ -0,0 +1,10 @@
+# IE11/Edge Browser Compatibility Considerations
+
+In order to properly support IE11 there are a few things to keep in mind:
+
+- Your app will need to provide a promise polyfill, such as **Bluebird**
+- We recommend using **Redirect flow** as there are situations where popups are blocked
+- We recommend setting `storeAuthStateInCookie` to true in your `msalConfig.cache` to prevent looping issues
+- Ensure your app and your **authority domain** (such as `login.microsoftonline.com`) are both added as **Trusted Sites** in the same **Security Zone**
```

Comment:
I would say something like "are both in the same security zone (i.e. "Trusted Sites" is recommended)"

Reply:
Got it, updated both ie11 sample READMEs